using-mobile-extensions/adobe-target/README.md

```diff
@@ -133,32 +165,62 @@ public class TargetApp extends Application {
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
+#### Swift
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {    
+  MobileCore.registerExtensions([Target.self, Identity.self]) {
+       //Completion callback
+       // Use the App id assigned to this application via Adobe Launch
+       MobileCore.configureWith(appId: "yourAppId")
+  }
+  return true
+}
+```
 #### Objective C
 
 In your app's `didFinishLaunchingWithOptions` function, register the Target extension with Mobile Core:
 
 ```objectivec
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-  [ACPCore configureWithAppId:@"yourAppId"];
-  [ACPTarget registerExtension];
-  [ACPIdentity registerExtension];
-  [ACPCore start:nil];
-  // Override point for customization after application launch.
-  return YES;
+    [AEPMobileCore registerExtensions: @[AEPMobileIdentity.class, AEPMobileTarget.class] completion:^{
+        //Completion callback
+        // Use the App id assigned to this application via Adobe Launch
+        [AEPMobileCore configureWithAppId: @"yourAppId"];
```

Comment:
`configureWithAppId:` is being invoked in two places now, remove duplicate API call on L193.